index.js

```diff
@@ -85,22 +85,38 @@ export const utils = {
   calloutIcons,
 };
 
+/**
+ * Pre-parse reusable content blocks. Note that they do not pass the content
+ * blocks recursively.
+ */
+const parseReusableContent = ({ reusableContent, ...opts }) => {
+  const parsedReusableContent = Object.entries(reusableContent).reduce((memo, [name, content]) => {
```

Comment:
Should we wrap this in a conditional that checks if the reusable content option is turned on?

Reply:
I was sort of imagining we just don't pass in any blocks. It won't do any extra parsing then.